src/constants/ticketdetailsconstant.js

```diff
@@ -1,41 +0,0 @@
-
-export const TEXTS = {
-    NAV_HEADING: "New Ticket",
-    DEVELOPER_LABEL: "Developer",
-    STATUS_LABEL: "Ticket Status",
-    PRODUCT_LABEL: "Ticket Product",
-    TYPE_LABEL: "Type",
```

Comment:
Why are we deleting this filee?

Reply:
As I moved all the constants into a single file (constants.js)  for better readability.